### DISCUSSION TITLE
General feedback for CTVBot v0.6

### DISCUSSION THREAD

**Discussion by jlplenio** [KevinBytesTheDust/CVAmp#119] [Announcements]
Please share your general positive and negative feedback (e.g., traffic, usability, system utilization, stability) here.
If you are sure you found a bug, [open a bug-report issue instead](https://github.com/jlplenio/crude-twitch-viewer-bot/issues/new/choose).

Add the following information to your post:
CTVBot version: e.g., 0.5.1
Operating system: e.g., Windows 11
Platform: e.g., Twitch.TV

**Comment by MrAdex77** [comment]
Hello,
 is there a possibility to add bin also for 64 bit system os? i wanted to test it on raspberry pi but i cannot open it with an error "cannot execude binary file: Exec format error"
 
CTVBot version: 0.5.3
Operating system:Linux Raspberry pi os 64bit

**Reply by jlplenio** [reply]
Assuming you want to run it on a Raspberry Pi, you need an ARM-based executable.
Github doesn't currently offer a native ARM runner and I have no idea about qemu (emulation).
But if they do in the future, I will easily add ARM-based runners for Raspberry Pi or Apple Silicon.

**Comment by SRZIsHere** [comment]
Great work, 

Maybe could you create a Discord for the project ?

And is that in your plan to add a chatbot feature ?

**Reply by jlplenio** [reply]
Sorry, I do not use Discord. Managing Github and ko-fi is enough.
For the chat-feature, you first need to log in. That is what I am working on atm here: https://github.com/jlplenio/crude-twitch-viewer-bot/discussions/163
